scipy/signal/signaltools.py

```diff
@@ -1593,6 +1595,10 @@ def correlate2d(in1, in2, mode='full', boundary='fill', fillvalue=0):
         A 2-dimensional array containing a subset of the discrete linear
         cross-correlation of `in1` with `in2`.
 
+    Notes
```

Comment:
Notes section is usually before the examples for a better readability.

Reply:
I think already the Notes section is "before" the examples. You mean "after"?

Reply:
Ah sorry, my bad. Got confused with the previous part. Nevermind